modules/apps/transfer/types/transfer_authorization.go

```diff
@@ -171,6 +176,21 @@ func isAllowedAddress(ctx sdk.Context, receiver string, allowedAddrs []string) b
 	return false
 }
 
+// areHopsAllowed returns whether the provided slice of Hop matches one of the allowed ones.
+func areHopsAllowed(provided []Hop, allowed []*Hops) bool {
```

Comment:
nit, maybe would go for something like `isAllowedForwarding`.. food for thought. Happy to hear input from others as well!

Reply:
I think that makes sense, more in line with the rest on the function